src/widget/wspinnyglsl.cpp

```diff
@@ -97,17 +97,17 @@ void WSpinnyGLSL::paintGL() {
     m_textureShader.bind();
 
     int matrixLocation = m_textureShader.matrixLocation();
-    int samplerLocation = m_textureShader.samplerLocation();
+    int textureLocation = m_textureShader.textureLocation();
     int positionLocation = m_textureShader.positionLocation();
     int texcoordLocation = m_textureShader.texcoordLocation();
 
-    QMatrix4x4 matrix;
+    QMatrix matrix;
```

Comment:
`QMatrix` is deprecated in Qt5 and removed in Qt6, why did you change to it?

Reply:
good question. i'll change it back.